integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/bootstrap/ElasticsearchBootstrapIT.java

```diff
@@ -127,7 +127,11 @@ public void noVersionCheck_incompleteVersion() {
 						" use the same model dialect." +
 						" It is not the case on ES 5 in particular, since 5.6 has a dialect" +
 						" but 5.0, 5.1, etc. don't have one.",
-				isAtMost( ElasticsearchTestDialect.getActualVersion(), "elastic:5.6" )
+				condition(
+						ElasticsearchTestDialect.getActualVersion(),
```

Comment:
If you're always using the version from `ElasticsearchTestDialect`, wouldn't it be simpler to expose a `condition` method in `ElasticsearchTestDialect` that implicitly uses that version?

I wonder if some renaming wouldn't also make the thing feel more smooth, e.g.:

```java
ElasticsearchTestDialect.isVersion(
		elasticsearch -> elasticsearch.isAtMost( "5.6" ),
		opensearch -> false
)
```

WDYT?